panel/src/components/Forms/Input/TagsInput.vue

```diff
@@ -318,7 +325,9 @@ export default {
       this.tags.splice(this.index(tag), 1);
       this.onInput();
 
-      if (prev) {
+      if (this.tags.length === 0) {
+        this.$refs.input.focus();
+      } else if (prev) {
```

Comment:
Focus to input if no tags remain

Reply:
Mmm something else is weird then. We should not need this. We test first if there is a previous tag, if there is a next tag and the last else would already focus the input. If no tags remain, `prev` and `next` should already not match and the `else` part should be applied.

Reply:
Because we get the previous and next neighbors before deleting tag, we have to make sure that no labels remain after deleting:

https://github.com/getkirby/kirby/blob/73de9f5e087157f992f2ca777feebe17f451c0dc/panel/src/components/Forms/Input/TagsInput.vue#L321-L322

Reply:
Yes, but if there was a previous tag before deleting, then the length can't be 0 (cause that prev tag should still be there, right).

Reply:
No, if there is a previous neighbor, there are already 2 tags. So length after delete is `1`  :)

Reply:
Exactly. And we already have

```js
if (pref) {
  …
} else if (next) {
  …
} else {
 // focus on input 
}
```

So I don't think we need your extra lines. These should already cover the case: If one is before, if one is after, else (none before, none after = 0 in length).

If this is not working, then something is wrong with getting `prev` and `next`

Reply:
I understand you're right. But there was an error in that part but I will test it again and share the details with you.